telegram/ext/jobqueue.py

```diff
@@ -408,6 +467,8 @@ def __init__(self,
         self._enabled = Event()
         self._enabled.set()
 
+        self.is_monthly = is_monthly if is_monthly is not None else False
```

Comment:
Please move this up and put it under `self.repeating`

Reply:
done